homeassistant/components/webhook/__init__.py

```diff
@@ -87,11 +87,21 @@ def async_generate_id() -> str:
 
 @callback
 @bind_hass
-def async_generate_url(hass: HomeAssistant, webhook_id: str) -> str:
+def async_generate_url(
+    hass: HomeAssistant,
+    webhook_id: str,
+    prefer_external: bool = True,
+    allow_ip: bool | None = None,
+) -> str:
     """Generate the full URL for a webhook_id."""
-    return (
-        f"{get_url(hass, prefer_external=True, allow_cloud=False)}"
-        f"{async_generate_path(webhook_id)}"
+    return "{}{}".format(
```

Comment:
I don't think "{}{}" is an improvement over the f-string. Maybe at least name the format parameters, `"{url}{path}".format(...)`?

Reply:
I believe this PR is so old now, it was changed in the meantime, because I don't see myself replacing f-strings with formats 😅